mpmp/src/cmds/SellPlot.java

```diff
@@ -0,0 +1,86 @@
+package cmds;
+
+import java.util.Arrays;
+import main.Client;
+import main.Conn;
+import main.ErrCode;
+import model.Player;
+import model.Plot;
+import view.Displayer;
+
+/**
+ * C->S
+ * @author Leander
+ */
+public class SellPlot implements CmdFunc{
+	private Displayer d;
+
+	@Override
+	public void exec(String line, Conn conn) {
+		String[] args = line.split(" ");
+		int price = -1;
+		
+		Player p = Player.search(((Client) conn).getName());
+		if (!p.isPlayer()) {
+			conn.sendErr(ErrCode.NotAPlayer);
+			return;
+		}
+		
+		if (args.length < 4) {
+			conn.sendErr(ErrCode.Usage, "sell-plot <Name des Grundstückes> <Käufer> <Preis>");
```

Comment:
My way is as follows: split the string, find first all-digits element (test via `String.matches("[0-9]+")`). That is the price, everything thereafter the name, everything before the plot.